### ISSUE TITLE
ABS try to load SearchView even if the MenuItem doesn't declare it

### ISSUE THREAD

**Issue by markov00** [JakeWharton/ActionBarSherlock#469]
Hello,
I'm expecting this error when I declare any menu item in my app. My menu xml is:

``` xml
<menu xmlns:android="http://schemas.android.com/apk/res/android" >
    <item
        android:id="@+id/ldf_menu_info"
        android:icon="@drawable/ic_menu_info_details"
        android:showAsAction="ifRoom"
        android:title="@string/ldf_menu_info">
    </item>
</menu>
```

But when the Activity inflate the menu I can see this error for any menu item in my menu, also if i didn't declare the SearchView as ActionView
I'm using the last version of ABS 4.0.2 and I'm compiling against API 15 with google maps support
Thanks

```
W/dalvikvm(22664): dvmFindClassByName rejecting 'res/drawable/textfield_searchview_holo_dark.xml'
W/MenuInflater(22664): Cannot instantiate class: res/drawable/textfield_searchview_holo_dark.xml
W/MenuInflater(22664): java.lang.ClassNotFoundException: res/drawable/textfield_searchview_holo_dark.xml in loader dalvik.system.PathClassLoader[/system/framework/com.google.android.maps.jar:/data/app/com.adnkronos.ldf-2.apk]
W/MenuInflater(22664):  at dalvik.system.PathClassLoader.findClass(PathClassLoader.java:251)
W/MenuInflater(22664):  at java.lang.ClassLoader.loadClass(ClassLoader.java:540)
W/MenuInflater(22664):  at java.lang.ClassLoader.loadClass(ClassLoader.java:500)
W/MenuInflater(22664):  at com.actionbarsherlock.view.MenuInflater$MenuState.newInstance(MenuInflater.java:463)
W/MenuInflater(22664):  at com.actionbarsherlock.view.MenuInflater$MenuState.readItem(MenuInflater.java:366)
W/MenuInflater(22664):  at com.actionbarsherlock.view.MenuInflater.parseMenu(MenuInflater.java:147)
W/MenuInflater(22664):  at com.actionbarsherlock.view.MenuInflater.inflate(MenuInflater.java:97)
W/MenuInflater(22664):  at com.adnkronos.ldf.activity.SezioniActivity.onCreateOptionsMenu(SezioniActivity.java:54)
W/MenuInflater(22664):  at com.actionbarsherlock.app.SherlockActivity.onCreatePanelMenu(SherlockActivity.java:167)
W/MenuInflater(22664):  at com.actionbarsherlock.ActionBarSherlock.callbackCreateOptionsMenu(ActionBarSherlock.java:542)
W/MenuInflater(22664):  at com.actionbarsherlock.internal.ActionBarSherlockCompat.preparePanel(ActionBarSherlockCompat.java:476)
W/MenuInflater(22664):  at com.actionbarsherlock.internal.ActionBarSherlockCompat.dispatchInvalidateOptionsMenu(ActionBarSherlockCompat.java:271)
W/MenuInflater(22664):  at com.actionbarsherlock.internal.ActionBarSherlockCompat$1.run(ActionBarSherlockCompat.java:981)
W/MenuInflater(22664):  at android.os.Handler.handleCallback(Handler.java:587)
W/MenuInflater(22664):  at android.os.Handler.dispatchMessage(Handler.java:92)
W/MenuInflater(22664):  at android.os.Looper.loop(Looper.java:132)
W/MenuInflater(22664):  at android.app.ActivityThread.main(ActivityThread.java:4123)
W/MenuInflater(22664):  at java.lang.reflect.Method.invokeNative(Native Method)
W/MenuInflater(22664):  at java.lang.reflect.Method.invoke(Method.java:491)
W/MenuInflater(22664):  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:841)
W/MenuInflater(22664):  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:599)
W/MenuInflater(22664):  at dalvik.system.NativeStart.main(Native Method)
```

**Comment by JakeWharton** [comment]
The textfield_searchview_holo_dark.xml layout is not part of the library.
Perhaps your R.java is stale. Try a clean build.
On May 12, 2012 10:12 AM, "Marco Vettorello" <
reply@reply.github.com>
wrote:

**Comment by markov00** [comment]
I tried (clean the sherlock build and also my project build), and I also create a brand new project with just only a demo activity and with one menuitem and the problem still.
The problem seems that the MenuInflater.MenuState.newInstance() wants to create a new instance of a wired class name: if I debug the classname that the menu inflater wants to instanciate is "res/drawable/textfield_searchview_holo_dark.xml"
It seems to happen only on my asus transformer with 3.2.1 but not with my nexus s 4.0.2

**Comment by JakeWharton** [comment]
Does the ASUS Transformer use stock Android or a custom ROM? It's probably a result of ASUS being jerks if they use a custom ROM.

**Comment by markov00** [comment]
The ASUS use a stock rom.  I will investigate with an older version of ABS, if this can be related.

**Comment by JakeWharton** [comment]
Fixed with #540.
